HaxeManual/02-types.tex

```diff
@@ -974,10 +974,10 @@ \subsection{Enum abstracts}
 
 \since{4.0.0}
 
-Since Haxe 4, it is possible to declare enum abstracts without using the \expr{:enum} metadata, instead using the more natural syntax \expr{enum abstract}. Additionally, if the underlying type is \expr{String} or \expr{Int}, the values for the enum cases can be omitted and are deduced by the compiler:
+Since Haxe 4, it is possible to declare enum abstracts without using the \expr{:enum} metadata, instead of using the more natural syntax \expr{enum abstract}. Additionally, if the underlying type is \expr{String} or \expr{Int}, the values for the enum cases can be omitted and are deduced by the compiler:
```

Comment:
No, this changes the meaning of the sentence. If the original sounds wrong, maybe `... metadata, using the more natural syntax \expr{enum abstract} instead.`